lib/shopify_api/rest/base.rb

```diff
@@ -17,6 +17,7 @@ class Base
       @paths = T.let([], T::Array[T::Hash[Symbol, T.any(T::Array[Symbol], String, Symbol)]])
       @custom_prefix = T.let(nil, T.nilable(String))
       @read_only_attributes = T.let([], T.nilable(T::Array[Symbol]))
+      @block_update_attributes = T.let([], T::Array[Symbol])
```

Comment:
nit: This kinda reads like, this blocks updates to this attribute.
But I don't have any better suggestions. So 🤷

Reply:
We could name this `update_attributes_as_block`!